src/overlays/actors/ovl_Bg_Last_Bwall/z_bg_last_bwall.c

```diff
@@ -28,57 +49,217 @@ const ActorInit Bg_Last_Bwall_InitVars = {
     (ActorFunc)BgLastBwall_Draw,
 };
 
-// static ColliderTrisElementInit sTrisElementsInit[2] = {
-static ColliderTrisElementInit D_80C189C0[2] = {
+static ColliderTrisElementInit sTrisElementsInit[] = {
     {
-        { ELEMTYPE_UNK0, { 0x00000000, 0x00, 0x00 }, { 0x00000008, 0x00, 0x00 }, TOUCH_NONE | TOUCH_SFX_NORMAL, BUMP_ON, OCELEM_NONE, },
+        {
+            ELEMTYPE_UNK0,
+            { 0x00000000, 0x00, 0x00 },
+            { 0x00000008, 0x00, 0x00 },
+            TOUCH_NONE | TOUCH_SFX_NORMAL,
+            BUMP_ON,
+            OCELEM_NONE,
+        },
         { { { -70.0f, 0.0f, 3.0f }, { 70.0f, 0.0f, 3.0f }, { -70.0f, 200.0f, 3.0f } } },
     },
     {
-        { ELEMTYPE_UNK0, { 0x00000000, 0x00, 0x00 }, { 0x00000008, 0x00, 0x00 }, TOUCH_NONE | TOUCH_SFX_NORMAL, BUMP_ON, OCELEM_NONE, },
+        {
+            ELEMTYPE_UNK0,
+            { 0x00000000, 0x00, 0x00 },
+            { 0x00000008, 0x00, 0x00 },
+            TOUCH_NONE | TOUCH_SFX_NORMAL,
+            BUMP_ON,
+            OCELEM_NONE,
+        },
         { { { 70.0f, 0.0f, 3.0f }, { 70.0f, 200.0f, 3.0f }, { -70.0f, 200.0f, 3.0f } } },
     },
 };
 
-// static ColliderTrisInit sTrisInit = {
-static ColliderTrisInit D_80C18A38 = {
-    { COLTYPE_NONE, AT_NONE, AC_ON | AC_TYPE_PLAYER, OC1_NONE, OC2_NONE, COLSHAPE_TRIS, },
-    ARRAY_COUNT(sTrisElementsInit), D_80C189C0, // sTrisElementsInit,
+static ColliderTrisInit sTrisInit = {
+    {
+        COLTYPE_NONE,
+        AT_NONE,
+        AC_ON | AC_TYPE_PLAYER,
+        OC1_NONE,
+        OC2_NONE,
+        COLSHAPE_TRIS,
+    },
+    ARRAY_COUNT(sTrisElementsInit),
+    sTrisElementsInit, // sTrisElementsInit,
+};
+
+static BgLastBwallModelInfo D_80C18A48[] = {
+    { object_last_obj_DL_0001A8, NULL, &object_last_obj_Colheader_000288 },
+    { object_last_obj_DL_001318, NULL, &object_last_obj_Colheader_001450 },
+};
+
+static Vec3s D_80C18A60[] = {
+    { 0x118, 0x118, -0x28 },
+    { 0x118, 0x118, 0x78 },
+    { 0x118, 0x190, 0x78 },
+    { 0x118, 0x190, -0x28 },
```

Comment:
This is used as a position offset (note it's converted into a Vec3f immediately) so should all go in dec (the numbers make more sense as dec anyway).